guidance/be-nice.md

```diff
@@ -0,0 +1,53 @@
+# Contributors Code of Conduct
+
+The .NET Foundation was created to foster an open, innovative and inclusive
+community around open source .NET. To clarify expected behaviour in our
```

Comment:
Should we also talk about providing a safe environment? Seems like the whole point here is that everyone can focus on the project without distractions and micro-aggression?